stripe.go

```diff
@@ -476,3 +476,31 @@ func initUserAgent() {
 	}
 	encodedStripeUserAgent = string(marshaled)
 }
+
+// Bool returns a pointer to the bool value passed in.
+func Bool(v bool) *bool {
+	return &v
+}
+
+// BoolValue returns the value of the bool pointer passed in or
+// false if the pointer is nil.
+func BoolValue(v *bool) bool {
```

Comment:
Just OOC, do you think that encapsulating this in a function is a good optimization? I wonder if handling `nil` a little more explicitly might make the code a little easier to understand. (And for clarity, I'm not particularly against it or anything.)

Reply:
Hmmm I just copy/pasted what the AWS SDK does (you referred me to it) and it seemed easier for most cases: https://github.com/aws/aws-sdk-go/blob/71a68668c287f9ad58d0c58c78e27f08eb39fa49/aws/convert_types.go#L64-L76

Reply:
Ah, k. I didn't know about the `*Value` functions in AWS. It's fine to leave them.